test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/015_OnVehicleData_notification_is_transferred_to_app_for_deprecated_fuelLevel_and_fuelLevel_State_params.lua

```diff
@@ -0,0 +1,32 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: SDL successfully transfers OnVehicleData notification for deprecated fuelLevel and fuelLevel_State
```

Comment:
Please remote the current script and extend the checks with OnVD in scripts with deprecated parameters(010. 011)